src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift

```diff
@@ -457,16 +459,44 @@ final class RiskProvider: RiskProviding {
 	
 	private func checkIfRiskLevelHasChangedForNotifications(_ risk: Risk) {
 		/// Triggers a notification for every risk level change.
-		if risk.riskLevelHasChanged {
```

Comment:
Could the code be cleaned to not use `risk.riskLevelHasChanged` anymore, but instead check if `risk.riskLevelChange` is not .unchanged? Asking, because I'm a little bit confused about the 2 properties now.

Reply:
I will have a look if we can refactor it to make `riskLevelHasChanged` a computed property